lib/plugins/aws/deploy/compile/events/apiGateway/README.md

```diff
@@ -120,5 +120,28 @@ functions:
                     identityValidationExpression: someRegex
 ```
 
+### Setting API keys for your Rest API
+You can specify a list of API keys to be used by your service Rest API by adding an `apiKeys` array property to the `provider` object in `serverless.yaml`. You'll also need to explicitly specify which endpoints are `private` and require one of the api keys to be included in the request by adding a `private` boolean property to the `http` event object you want to set as private.
+
+Here's an example configuration for setting API keys for your service Rest API:
+
+```yaml
+service: my-service
+provider:
+    name: aws
+    apiKeys:
+            - myFirstKey
+            - ${mySecondSecretKey} # you can hide it in a serverless variable
+functions:
+    hello:
```

Comment:
There was recently a switch from 4 level indentation to two levels. Could you update the indentation?